functions/getEmployees/sample.dat

```diff
@@ -0,0 +1,3 @@
+{

```

Comment:
Is this needed?

Reply:
Probably not, i could make proper dummy data. But i have never experienced someone using it 🤔